### PR TITLE
MinVar

### PR DESCRIPTION
**Pull request by ozagordi**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by ozagordi** [comment]
Hi
I apologise for the multiple commits and builds. This now works, provided that one has installed `lofreq` independently. The version on bioconda of `lofreq` only works with python 2.7, there is a related PR #4950.

File: recipes/minvar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package:
+  name: minvar 
+  version: "1.2a3"
+
+build:
+  number: 0
+  skip: True # [not py35]
```
Comment (bgruening) [inline recipes/minvar/meta.yaml]: is this skip really needed? builds for py36 seems to pass

File: recipes/minvar/licence.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+SOFTWARE LICENSE AGREEMENT (hereinafter referred to as AGREEMENT)
+=================================================================
+
+This is a legal agreement between you (hereinafter referred to as the LICENSEE)
+and the University of Zurich, Rämistrasse 71, 8006 Zurich (hereinafter referred
+to as UNIVERSITY OF ZURICH) pertaining to the right to use the software product
+“MinVar” (hereinafter referred to as SOFTWARE). Using the SOFTWARE indicates
+your acceptance of these terms and conclusion of the Agreement between the
```
Comment (bgruening) [inline recipes/minvar/licence.md]: usually we do not ship license files, is there any reason for it?

Comment (ozagordi) [inline recipes/minvar/licence.md]: Right. Licence is custom and there is a licence file field, I thought to fill it but then I forgot. Is it ok if I remove `licence.md` and delete the `licence_file` field?

File: recipes/minvar/meta.yaml (hunk 1 above)
Comment (ozagordi) [inline recipes/minvar/meta.yaml]: Ok, then I will skip py27 only.

File: recipes/minvar/licence.md (hunk 2 above)
Comment (bgruening) [inline recipes/minvar/licence.md]: This sounds alright, but normally this license file is part of the tarball/package and license_file with copy it by them self. So no need to put it into the repo.

File: recipes/minvar/meta.yaml (hunk 1 above)
Comment (bgruening) [inline recipes/minvar/meta.yaml]: Because of the lofreq dependency? Can you wait until this PR is merged?

Comment (ozagordi) [inline recipes/minvar/meta.yaml]: Sure. At least at the beginning, though, I will still skip py27. I haven't written minvar with support for it. What is your policy for tools that depend on something that is not (yet) on bioconda?

**Comment by ozagordi** [comment]
@bgruening, besides py27 (python 2 is not supported) I had to skip py36, build failed otherwise.

**Comment by bgruening** [comment]
Any idea why py3k failed? Is this on our side?

**Comment by bgruening** [comment]
Can you add this reason as a comment to the skip section.

**Comment by bgruening** [comment]
Build number should start with 0. Thanks!

**Comment by ozagordi** [comment]
Hi @bgruening.
py36 fails because setuptools_scm_git_archive is not available. I added this in a comment and reset the build to 0.

Thanks!

**Comment by bgruening** [comment]
@ozagordi sorry for the slow turnaround, this usually is faster but a lot of us are currently on holiday.
One last question, why do you need scm_git, are you downloading packages that should be a dependency?

**Comment by ozagordi** [comment]
No, `setuptools_scm_git_archive` is a plugin that allows the automatic parsing of the release version from GitHub. As a developer I don't need to manually change the value of a variable every time that I make a new release, the user runs `minvar --version` and the version is always correct.

**Comment by bgruening** [comment]
Thanks @ozagordi! :)

**Pull request merged**